README.md

```diff
@@ -54,4 +54,4 @@ This Repository contains everything you should know to become an awesome web dev
 
 ## Projects
 1. Gym Website
-2. On;line Meal Delivery Website
+2. Online Meal Delivery Website
```

Comment:
It is already corrected in previous pull request.

Reply:
Actually there is a semicolon

Reply:
yes but it is corrected in #3 .